docs/idl_vs_python/python_vs_IDL_fit.rst

```diff
@@ -0,0 +1,48 @@
+###############################
+IDL vs. Python Spectrum Fitting
+###############################
+
+Fit Results
+===========
+
+The surface brightness data in the following spectra are from a southern region of 
+NGC 2023. To learn more about the region of NGC 2023 examined, refer to *The PAH 
+Emission Characteristics of the Reflection Nebula NGC 2023* by Els Peeters et al.
```

Comment:
The full reference including journal, number, page should be given.  In addition, a link to the paper record on ADS should be added.